tests/foreman/cli/test_auth.py

```diff
@@ -188,21 +187,21 @@ def test_positive_change_session(admin_user, non_admin_user):
     :expectedresults: The session is altered
 
     """
-    result = configure_sessions()
+    result = configure_sessions(default_sat)
     assert result == 0, 'Failed to configure hammer sessions'
     AuthLogin.basic({'username': admin_user['login'], 'password': password})
     result = Auth.with_user().status()
     assert LOGEDIN_MSG.format(admin_user['login']) in result[0]['message']
     # list organizations without supplying credentials
-    assert Org.with_user().list()
+    assert User.with_user().list()
```

Comment:
@omkarkhatavkar  Is this an expected change ?

Reply:
Yes it is expected change, I was not able to comment in the code. Basically the org acess is limited for other users.